### DISCUSSION TITLE
Add global open/close methods for modals

### DISCUSSION THREAD

**Discussion by ErriourMe** [tailwindlabs/headlessui#1998] [Ideas]
I colud open modal by id from any place through [global methods](https://bootstrap-vue.org/docs/components/modal#using-thisbvmodalshow-and-thisbvmodalhide-instance-methods)  in Bootstrap-Vue (Nuxt). For example, `$bvModal.show('myModal')`. It's really helpful when I have modal and I want open it twice in several places (for example, login modal in header (overlay component) and in comments section (component in article page)).

Is it possible here? Or can be it added?

**Comment by mcube27** [comment]
I have the same exact case
